### PR TITLE
Ik/replace some entities

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

File: mmdet/apis/ote/apis/detection/config_utils.py [hunk 1]
```diff
@@ -14,23 +14,26 @@
...
-from sc_sdk.entities.label import Label
-from sc_sdk.logging import logger_factory
+
+from mmdet.apis.ote.extension.utils.logging import get_logger
 
 from .configuration import OTEDetectionConfig
 
-logger = logger_factory.get_logger("OTEDetectionTask")
+
+logger = get_logger("server.OTEDetectionTask")
```
Comment (druzhkov-paul) [inline mmdet/apis/ote/apis/detection/config_utils.py]: What does "server" mean?

File: mmdet/apis/ote/apis/detection/openvino_task.py [hunk 2]
```diff
@@ -53,7 +53,8 @@
 
 from .configuration import OTEDetectionConfig
 
-logger = logger_factory.get_logger("OTEDetectionTask")
+
+logger = get_logger("server.OTEDetectionTask.Hooks")
```
Comment (druzhkov-paul) [inline mmdet/apis/ote/apis/detection/openvino_task.py]: Please fix the logger name.

File: mmdet/apis/ote/apis/detection/config_utils.py (hunk 1 above)
Comment (Ilya-Krylov) [reply mmdet/apis/ote/apis/detection/config_utils.py]: removed, used std logging

File: mmdet/apis/ote/apis/detection/openvino_task.py (hunk 2 above)
Comment (Ilya-Krylov) [reply mmdet/apis/ote/apis/detection/openvino_task.py]: fixed

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**